updating python mondemand with performance traces and annotations

luke-powers opened the pull request:
Also building mondemand python client with newer version of swig (3.0.10 vs 2.0.7)

djnym commented:
Do you think you can update the demo.py and add a README.md with instructions on how to build and use?

djnym commented:
I still think that demo.py should be changed to show that this works (maybe with instructions to run lwes-event-printing-listener and run demo.py to see that the messages all go through).  Also demo.py at least gives an example of how to use the library.

luke-powers commented:
Updated the make file (and setup.py) to properly install when `make; make install` is run. Updated the README with examples of how to use the demo.py with lwes-event-listener.
